mkosi/__init__.py

```diff
@@ -2255,3 +2260,13 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
 
             if args.verb == Verb.serve:
                 run_serve(last)
+
+
+def normalize_mtime(source_date_epoch: Optional[int], root: Path) -> None:
```

Comment:
Please pass `root` as the first argument for consistency with our other functions. Also pass in the directory in `root` to operate on as a separate argument. 

Also please rename the `source_date_epoch` argument to `mtime`